README.md

```diff
@@ -60,11 +61,21 @@ GStreamer is installed on your system.
 By default we download all the libraries from GitHub and build them locally, so should require nothing to be installed ahead of time.
 If you do wish to link to existing libraries you can use the following flags to customize your build.
 
+
+#### Cross-Compilation
+
+If you wish to cross-compile `CC` and `CXX` are respected when building the library and all its dependencies. You will also need to set `BUILD_OPENSSL_PLATFORM`, `BUILD_LIBSRTP_HOST_PLATFORM` and `BUILD_LIBSRTP_DESTINATION_PLATFORM`. See our [.travis.yml](.travis.yml) for an example of this. Every commit is cross compiled to ensure that it continues to work.
```

Comment:
are we saying that the following environment variables are to be defined properly to match the toolchain settings.
CC, CXX ... BUILD_LIBSRTP_DESTINATION_PLATFORM

Reply:
`CC` and `CXX` need to be set.

`BUILD_LIBSRTP_DESTINATION_PLATFORM` are CMake arguments. Should I reword the documentation or maybe update the README in other ways?

thanks